data/json/items/migration.json

```diff
@@ -1313,5 +1313,55 @@
     "id": "rifle_3006",
     "type": "MIGRATION",
     "replace": "remington_700"
+  },
+  {
+    "id": "balclava",
+    "type": "MIGRATION",
+    "replace": "balaclava"
```

Comment:
Not sure if it was discussed before, should there also be comments (`"\\"` JSON key) about version the migration was added in? In this case, `0.F`.

Reply:
1. I don't see any of the other migrations having this comments
2. Isn't git blame sufficient?

Reply:
1. It's just a suggestion that came to my mind minutes ago :)
2. Definitely sufficient but makes determining what migrations should be dropped for which version harder and can be made even harder if JSON style ever changes and it's re-linted or something similar.

Reply:
Anyway, feel free to resolve this without changes, I don't mind